frontend/src/loadPostHogJS.tsx

```diff
@@ -22,9 +22,7 @@ export function loadPostHogJS(): void {
             window.JS_POSTHOG_API_KEY,
             configWithSentry({
                 api_host: window.JS_POSTHOG_HOST,
-                _capture_metrics: true,
                 rageclick: true,
-                debug: window.JS_POSTHOG_SELF_CAPTURE,
```

Comment:
This might be sometimes useful locally, but it does make a lot of noise, and thus hide valuable messages.